packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.js

```diff
@@ -69,7 +69,7 @@ function DefaultArrayTemplate(props) {
 							id={id && `${id}-control-${index}`}
 							index={index}
 							onRemove={onRemove}
-							onReorder={canReorder && onReorder}
+							onReorder={canReorder ? onReorder : undefined}
```

Comment:
here, before fix, onReorder can be false and trigger proptype warning since it should be a function

Reply:
```suggestion
							onReorder={canReorder ? onReorder : undefined}
```

Understand! Last suggestion, maybe a noop from lodash? Instead, you can have some TypeError like `undefined is not a function`

Reply:
Actually if onReorder is undefined, it won't get called because:
https://github.com/Talend/ui/blob/master/packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js#L108
it won't bother to render a reorder button.
But if I pass a noop, it will show a reorder button and click it won't do anything.
I prefer not to show a reorder button at all. WDYT? :)

Reply:
You'r right, thanks for the explanation 👍